lib/rolodex/config.ex

```diff
@@ -144,11 +144,13 @@ defmodule Rolodex.PipelineConfig do
 
   ## Example
 
+  ```
```

Comment:
This is pretty nit, but rather than wrap in triple backticks could we just double-indent the whole code block? That's what we do elsewhere (and I think what hexdocs recommends).